packages/snap/src/config/networks.ts

```diff
@@ -0,0 +1,61 @@
+import type { Chain } from 'viem';
+
+export const reticulumChain: Chain = {
```

Comment:
What do we need those configs for? Viem does not seem to be used anywhere in the snap.